spec/asyncapi.md

```diff
@@ -165,17 +165,17 @@ This object can be extended with [Specification Extensions](#specificationExtens
 
 #### <a name="A2SVersionString"></a>AsyncAPI Version String
 
-The version string signifies the version of the AsyncAPI Specification that the document complies to.
+The version string signifies the version of the AsyncAPI Specification that the document complies with.
 The format for this string _must_ be `major`.`minor`.`patch`.  The `patch` _may_ be suffixed by a hyphen and extra alphanumeric characters.
 
-A `major`.`minor` shall be used to designate the AsyncAPI Specification version, and will be considered compatible with the AsyncAPI Specification specified by that `major`.`minor` version.
+A `major`.`minor` shall be used to designate the AsyncAPI Specification version and be considered compatible with the AsyncAPI Specification specified by that `major`.`minor` version.
 The patch version will not be considered by tooling, making no distinction between `1.0.0` and `1.0.1`.
 
 In subsequent versions of the AsyncAPI Specification, care will be given such that increments of the `minor` version should not interfere with operations of tooling developed to a lower minor version. Thus a hypothetical `1.1.0` specification should be usable with tooling designed for `1.0.0`.
 
 #### <a name="A2SIdString"></a>Identifier
 
-This field represents a unique universal identifier of the [application](#definitionsApplication) the AsyncAPI document is defining. It must conform to the URI format, according to [RFC3986](https://tools.ietf.org/html/rfc3986).
+This field represents a unique universal identifier of the [application](#definitionsApplication) the AsyncAPI document defines. It must conform to the URI format, according to [RFC3986](http://tools.ietf.org/html/rfc3986).
```

Comment:
Why change https://tools.ietf.org/html/rfc3986 to http://tools.ietf.org/html/rfc3986 ?

I know the http URL redirects to the https one anyway, so it doesn't really matter - but I still think it's better to treat the https one as canonical and link to that.